.github/workflows/discord-discussions.yml

```
@@ -2,45 +2,213 @@ name: Discord Discussion Notification
 
 on:
   discussion:
+    types: [created, answered, unanswered, closed, reopened, labeled, unlabeled, category_changed]
```

actionlint (1.7.12)

[error] invalid activity type "closed" for "discussion" Webhook event. available types are "answered", "category_changed", "created", "deleted", "edited", "labeled", "locked", "pinned", "transferred", "unanswered", "unlabeled", "unlocked", "unpinned"

(events)